contracts/amm/spread/SpreadStorageLibs.sol

```diff
@@ -0,0 +1,147 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.16;
+
+import "@openzeppelin/contracts/utils/math/SafeCast.sol";
+
+import "../../libraries/math/IporMath.sol";
+import "forge-std/Test.sol";
+import "./SpreadTypes.sol";
+
+library SpreadStorageLibs {
+    using SafeCast for uint256;
+    uint256 private constant STORAGE_SLOT_BASE = 10_000;
+
+    /// Only allowed to append new value to the end of the enum
+    enum StorageId {
+        Owner,
+        AppointedOwner,
+        Paused,
+        WeightedNotional28DaysDai,
```

Comment:
Describe in details how are organized params in this particular slot (weightedNotionalPayFixed, lastUpdateTimePayFixed, weightedNotionalReceiveFixed)